addon/components/o-s-s/attribute/country.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="oss-attribute"
+     {{on "mouseenter" (fn (mut this.displayCopyBtn) true)}}
+     {{on "mouseleave" (fn (mut this.displayCopyBtn) false)}}
+     data-control-name="attribute-country" ...attributes>
```

Comment:
should we really put data-control-name here instead of letting the "context" set one ? maybe in the page it will be used it will more like "influencer-country" or "shipping-address-country" ?

Reply:
I was thinking better to have a default one at overwrite it if need be, no ? (unless we can't overwrite with the splattributes ? ^^)